components/profileImageComponents/ProfileImageAndName.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  ImageStyle,
+  StyleProp,
+  StyleSheet,
+  View,
+  ViewStyle
+} from "react-native"
+import { BodyText, Caption, Headline } from "../Text"
+import ProfileImage from "./ProfileImage"
+import React, { useState } from "react"
+import { Ionicons } from "@expo/vector-icons"
+import { Ionicon } from "@components/common/Icons"
+import { AppStyles } from "@lib/AppColorStyle"
+import Toast from "react-native-root-toast"
+
+interface ImageAndNameProps {
+  displayFriend?: boolean
+  displayHandle?: boolean
+  style?: StyleProp<ViewStyle>
+  imageStyle?: StyleProp<ImageStyle>
+  username: string
+  userHandle: string
+  eventColor: string
+  toastOffset?: number
+}
```

Comment:
Make sure that we can take in the friend status as a type so that this becomes reusable on the attendees list screen. That type could look something like this:

```ts
// Don't know if we need blocked but we definitely need the first 3.
type UserFriendStatus = "not-friends" | "friend-request-pending" | "friends" | "blocked"
```

Also it looks like we're starting to get quite a few flag arguments here, maybe we could have multiple components of the different variants?